pkg/deps/golang.go

```diff
@@ -119,7 +125,10 @@ func (resolver *GoModResolver) ResolvePackageLicense(module *packages.Module, re
 			}
 			identifier, err := license.Identify(module.Path, string(content))
 			if err != nil {
-				return err
+				if declareLicense == nil {
```

Comment:
If the users already declare this license I don't think we need to `license.Identify` it, this gives the users ability to override the license if we make something wrong

Reply:
Updated.